adapters/README.md

```diff
@@ -0,0 +1,32 @@
+# DBSP application server demo
```

Comment:
This repository is not a demo. I think this could be a better description. Ideally we'd use 

# DBSP I/O adapter framework

This crate implements an infrastructure to ingest data into a DBSP circuit from external
data sources and to stream the outputs of the circuit to external consumers. It also implements
a DBSP I/O controller that controls the execution of a DBSP circuit along with its input and output adapters, and a DBSP server that exposes the controller API over HTTP and through a web interface.

Reply:
it does contain a demo. I will update this.

Reply:
I will merge, but please check the new README